package.json

```diff
@@ -5,44 +5,33 @@
   "version": "1.0.0",
   "private": "true",
   "workspaces": [
-    "src/addons/volto-bise",
-    "src/addons/volto-n2k",
-    "src/addons/volto-slate",
-    "src/addons/volto-datablocks",
-    "src/addons/volto-plotlycharts",
-    "src/addons/volto-object-widget",
-    "src/addons/volto-columns-block",
-    "src/addons/volto-slate-footnote",
-    "src/addons/volto-block-style",
-    "src/addons/volto-webmap",
-    "src/addons/volto-matomo",
-    "src/addons/volto-searchkit",
-    "src/addons/volto-blocks-form"
+    "src/addons/*"
   ],
   "addons": [
+    "volto-bise",
+    "@eeacms/volto-n2k",
     "volto-slate:asDefault",
-    "@eeacms/volto-object-widget",
-    "volto-embed",
     "volto-datablocks",
     "volto-plotlycharts",
+    "@eeacms/volto-object-widget",
+    "@eeacms/volto-columns-block",
     "@eeacms/volto-slate-footnote",
+    "@eeacms/volto-block-style",
+    "@eeacms/volto-webmap",
+    "@eeacms/volto-matomo",
+    "volto-searchkit",
+    "@eeacms/volto-blocks-form",
+    "volto-embed",
     "@eeacms/volto-corsproxy",
-    "@eeacms/volto-columns-block",
-    "volto-bise",
-    "@eeacms/volto-n2k",
     "@eeacms/volto-slate-dataentity",
-    "@eeacms/volto-webmap",
     "@eeacms/volto-block-divider",
     "@eeacms/volto-block-toc",
     "@eeacms/volto-slate-metadata-mentions",
     "@eeacms/volto-widgets-view",
     "@eeacms/volto-taxonomy",
     "@eeacms/volto-middleware-rdf",
-    "@eeacms/volto-matomo",
-    "volto-searchkit",
-    "@eeacms/volto-blocks-form",
     "@eeacms/volto-sentry-rancher-config",
-    "@eeacms/volto-block-style"
```

Comment:
The order of addons is important, you shouldn't order them alphabetically. For example, for the volto-block-style, it's extremely important that it comes last. Also, another example: volto-bise, we treat it at the same level as project, it customizes some other addons, so it should also load as last as possible.

Reply:
Is the order good now? Locally BISE shows well inside the browser at the first look.